src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md

```diff
@@ -175,6 +180,73 @@ In the _Settings tab_ click _"Reveal Config Vars"_ button to see the
 applications environment variables. Click edit to update them or add new
 ones if needed.
 
+Check at least these variables:
+
```

Comment:
Extra space

Reply:
Removed the second (unintentional) line break.